Pipfile

```diff
@@ -85,6 +85,7 @@ spdx-tools = ">=0.8.0,<0.9.0"
 license-expression = ">=30.1.0,<31.0.0"
 rustworkx = ">=0.13.0,<1.0.0"
 pydantic = ">=2.0.0,<3.0.0"
+asteval = "==1.0.6"
```

Comment:
Note that it seems like this version doesn't support python3.8, which is still supported in checkov. Try to find a version which still has support for it